web/src/features/hfiCalculator/components/MeanIntensityGroupRollup.tsx

```diff
@@ -47,6 +47,9 @@ const useStyles = makeStyles({
   alignErrorIcon: {
     marginTop: '6px',
     textAlign: 'center'
+  },
+  default: {
+    background: 'rgba(40, 53, 147, 0.05)'
```

Comment:
I have an urge to want to start creating a colour palette file somewhere.... just a thought.